docs/user-management.md

```diff
@@ -0,0 +1,88 @@
+# BC Gov's Product Registry - User management documentation
+Last updated: **March 12, 2024**
+
+Welcome to the BC Gov's Product Registry documentation. This guide provides detailed information about the new user interface changes and the enhanced user management feature available at [BC Gov's Product Registry](https://registry.developer.gov.bc.ca/). With the latest update, users can now manage AWS permissions directly from the website, enhancing the overall user experience and streamlining project management.
+
+## New user interface 
+
+The BC Gov's Product Registry has undergone significant UI improvements to provide a more intuitive and user-friendly experience. You can now easily navigate through the website, access different sections, and manage your projects and users more efficiently.
+
+## User management feature
+
+One of the key updates is the introduction of a comprehensive user management feature. This feature allows product owners and technical leads to manage AWS permissions for users associated with their product. Users can be assigned to one or more of the following roles:
+
+1. **Viewers**
+2. **Billing viewers**
+3. **Developers**
+4. **Admins**
+5. **Security auditors**
+
+![New role management feature in the product registry](images/user-management/registry_roles.png)
```

Comment:
@Pilargit12 Hi Pilar, the text in the square brackets is actually the image alt text, not the file name. Would you like me to change what is there currently based on your suggestions?

Reply:
You are right I had missed that sorry!

Reply:
Just going to close this, change the alt text and create a new PR dev -> main